internal/transport/http2_client.go

```diff
@@ -1332,6 +1332,7 @@ func (t *http2Client) keepalive() {
 					timer.Reset(t.kp.Time)
 					continue
 				}
+				infof("transport: closing client transport due to keepalive.")
```

Comment:
"due to idleness / lack of connection activity"?

Reply:
Done